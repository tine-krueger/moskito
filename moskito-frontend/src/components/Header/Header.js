import styled from 'styled-components/macro'

export default function Header({children}) {
    return (
        <HeaderStyled>
            <h1>{children}</h1>
        </HeaderStyled>
    )
}

const HeaderStyled = styled.div`
    width: 80%;
    margin: 2em auto 3em;
    padding: .25em .5em;
    border-radius: 20px;
    text-align: center;
    box-shadow: 2px 4px 8px -4px rgba(0, 0, 0, 0.6), 
                -2px -2px 2px 0px rgba(255, 255, 255, 0.5);
`