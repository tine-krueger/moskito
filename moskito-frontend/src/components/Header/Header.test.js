import { render } from '@testing-library/react'
import 'jest-styled-components'
import Header from './Header'

describe('Header', () => {
  it('renders correctly', () => {
    const { container, getByText } = render(<Header>Your Favourites</Header>)
    expect(container.firstChild).toMatchSnapshot()
    expect(getByText(/Your Favourites/i)).toBeInTheDocument()
  })
})