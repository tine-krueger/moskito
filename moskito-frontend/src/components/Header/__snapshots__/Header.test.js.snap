// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Header renders correctly 1`] = `
.c0 {
  width: 80%;
  margin: 1em auto;
  padding: .25em .5em;
  border-radius: 20px;
  text-align: center;
  box-shadow: 2px 4px 8px -4px rgba(0,0,0,0.6),-2px -2px 2px 0px rgba(255,255,255,0.5);
}

<div
  class="c0"
>
  <h1>
    Your Favourites
  </h1>
</div>
`;
